src/components/UserData.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+UserData.propTypes = {
+  personalUserData: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      phone: PropTypes.string.isRequired,
+      mail: PropTypes.string.isRequired,
+      department: PropTypes.string.isRequired,
+    })
+  ),
+  skills: PropTypes.arrayOf(
+    PropTypes.shape({
+      skill1: PropTypes.string.isRequired,
+      skill2: PropTypes.string,
+      skill3: PropTypes.string,
+      skill4: PropTypes.string,
+      skill5: PropTypes.string,
+      skill6: PropTypes.string,
+    })
+  ),
+}
+
+export default function UserData({ personalUserData, skills }) {
+  return (
+    <DataWrapper>
+      <UserDates>
```

Comment:
Again, the name is a bit off. It suggests it has something to do with dates? `Details` maybe?